queries.sql

```diff
@@ -33,4 +33,76 @@ WHERE NOT name = 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 and 17.3;
+-- DAY 2
+-- Update the animals table by setting the species column to unspecified.
+-- Verify that change was made. Then roll back the change
+-- Verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+TABLE animals;
+ROLLBACK;
+TABLE animals;
```

Comment:
- [ ] Please kindly ensure to include queries to verify that the change was made before and after rollback. In this case, you can do something like this
`SELECT species from animals; `

Reply:
@AnselemOdims TABLE animals;
prints the entire table with the new updates, I used that as my verification query.
isn't it valid?